packages/server/src/tests/epics/epicSelection.ts

```diff
@@ -84,11 +84,8 @@ export const excludeTags: Filter = {
             return true;
         }
 
-        const intersection = test.excludedTagIds.filter(
-            tagId => !targeting.tags.map(tag => tag.id).includes(tagId),
-        );
-
-        return intersection.length > 0;
```

Comment:
Just trying to wrap my head around the existing logic. If the `intersection` has one or more elements, we should filter out the article, but here we would return `true`, which would keep the article

Reply:
yes this should be <= 0, but I think the `!` on line 88 is also wrong?
Anyway, I find the new logic easier to reason about!

Reply:
Yeah that's right! Flipping both of those should fix it, but agree the new logic is easier to follow!